docs/pagination.md

```diff
@@ -28,7 +28,9 @@ We can refetch the query by adding the same request to the `requestController`:
 client.requestController.add(reviewsReq);
 ```
 
-Once the response is received, any `request()` Streams that were listening to `reviewsReq` will get updated with the new data.
+Once the response is received, any `request()` Streams that were listening to `reviewsReq` will get updated with the new data. 
+
+Note that refetching don't override `FetchPolicy` setting, so if `FetchPolicy` is set to `CacheFirst`, you'll get the same results of the first operation.
```

Comment:
Thanks for this.

There are a couple grammatical nitpicks, and I'd also wrap it in a note block. Can you update it as follows?

```
:::note
Since refetching doesn't override the `FetchPolicy` setting, if `FetchPolicy` is set to `CacheFirst`, you'll get the cached results of the initial operation.
:::
```

Reply:
Thanks for corrections!!

Commited: https://github.com/gql-dart/ferry-docs/pull/2/commits/0cf17e0e63556b971f24208ceb3b4622e7af2663